core/dataflow/src/main/java/edu/uci/ics/texera/dataflow/common/PropertyNameConstants.java

```diff
@@ -115,6 +115,9 @@
     public static final String MYSQL_PASSWORD = "password";
     public static final String MYSQL_LIMIT = "limit";
     public static final String MYSQL_OFFSET = "offset";
+    
+    // related to Twitter converter
+    public static final String TWITTER_CONVERTER_RAW_JSON = "rawJsonStringField";
```

Comment:
We are calling fields as attribute when showing options to users. So, we should rename this as rawJsonStringAttributeName/rawJsonStringAttribute.

Reply:
Fixed, new name is `rawJsonStringAttributeName`